cmd/nerdctl/image/image_list_test.go

```diff
@@ -317,3 +317,39 @@ CMD ["echo", "nerdctl-build-notag-string"]
 
 	testCase.Run(t)
 }
+
+func TestImagesKubeWithKubeHideDupe(t *testing.T) {
+	nerdtest.Setup()
+
+	testCase := &test.Case{
+		Require: test.Require(
+			nerdtest.OnlyKubernetes,
+			test.Not(nerdtest.Docker),
```

Comment:
You don't need to `test.Not(nerdtest.Docker)`.
Because docker is never ran inside a Kube context, `nerdtest.OnlyKubernetes` will always be false.

(btw, I am not even sure how to use Docker against containerd in kube, but that is a different, albeit interesting, question)